stories/atoms/components/textField/TextField.stories.mdx

```diff
@@ -267,12 +267,23 @@ The `withBorder` property allows you to add a themed border to the textField and
 ## Multiline
 
 The `multiline` property allows you to write text on several lines. The simple text field becomes a text area.  
-When `multiline` is `true` and [disableAreaResize](/docs/atoms-textfield--disable-area-resize) is `false` the [size](/docs/atoms-textfield--sizes) is automatically `auto`.
+When `multiline` is `true` and [disableAreaResize](/docs/atoms-textfield--disable-area-resize) is `false`
+the [size](/docs/atoms-textfield--sizes) is automatically `auto`.  
+When `fullWidth` is `true`and [disableAreaResize](/docs/atoms-textfield--disable-area-resize) is `false`
```

Comment:
The description is not consistent here and as such can lead to confusion as to if `multiline` is necessary for the behavior. I suggest reworking the description so that we don't need to repeat the `multiline` **value**.

On a side note be aware that properties are supposed to be in bac tics and not hyperlink text.